doc/pprof.md

```diff
@@ -235,10 +239,41 @@ them. This is useful to combine profiles from multiple processes of a
 distributed job. The profiles may be from different programs but must be
 compatible (for example, CPU profiles cannot be combined with heap profiles).
 
-pprof can subtract a profile from another in order to compare them. For that,
-use the **-base= _profile_** option, where *profile* is the filename or URL for the
-profile to be subtracted. This may result on some report entries having negative
-values.
+
+## Comparing profiles
+
+pprof can subtract a profile from another, provided the profiles are of 
```

Comment:
Maybe start this section with a very high-level note that there are two ways to compare profiles, briefly characterize each of them, then talk about each in more detail.